JSDemos/Demos/Tabs/Overview/Angular/app/app.component.css

```diff
@@ -1,26 +1,107 @@
-::ng-deep .content {
-  text-align: justify;
-  margin-top: 25px;
+::ng-deep body {
+  margin: 0;
+  padding: 0;
 }
 
-::ng-deep #longtabs {
-  margin-top: 20px;
+::ng-deep #tabs-demo {
+  padding: 20px;
+  display: flex;
+  min-height: 450px;
+  justify-content: space-between;
 }
 
-::ng-deep #scrolledtabs {
-  margin-top: 20px;
+::ng-deep .first-group {
+  display: none;
+}
+
+::ng-deep .widget-container {
+  flex-grow: 1;
+  margin: 0 auto;
```

Comment:
You shouldn't use margin: 0 auto, if you use display: flex. Use align-items: center.